File: palindromo.js
function pal(str) {
    const strn = str.replace(/[\W_]/g, "")
                    .toLowerCase();
    const revertido = strn.split("")
                    .reverse()
                    .join("");

    return strn == revertido ? "es palindromo" : "no es palindromo";
}
console.log(pal("hola mundo")); 

File: palindromo.py
a='lool'
cont=len(a)
b=[]
for i in range(cont - 1, -1, -1):
    b.append(a[i])
revertido = "".join(b)
print(a + " es: ", end="")
if a == revertido:
    print("Es palindromo:"+revertido)
else:
    print("No es palindromo:"+revertido)